stablehlo/dialect/VhloTypes.cpp

```diff
@@ -120,6 +120,18 @@ void VhloTypeConverter::addBuiltinToVhloConversions() {
         convertedExpressedType, APFloat(type.getScale()), type.getZeroPoint(),
         type.getStorageTypeMin(), type.getStorageTypeMax());
   });
+  addConversion([&](quant::UniformQuantizedPerAxisType type) -> Type {
+    Type convertedStorageType = convertType(type.getStorageType());
+    Type convertedExpressedType = convertType(type.getExpressedType());
+    if (!convertedStorageType || !convertedExpressedType) return {};
+    SmallVector<APFloat> scales = llvm::to_vector(llvm::map_range(
```

Comment:
Why is this handling different from ZP handling?

Reply:
Actually, the `vhlo::UniformQuantizedPerAxisV1Type` takes APFloat for the scales whereas the `mlir::quant::UniformQuantizedPerAxisType` takes double.